src/output-json.c

```diff
@@ -410,7 +410,11 @@ void JsonPacket(const Packet *p, json_t *js, unsigned long max_length)
     unsigned long max_len = max_length == 0 ? GET_PKT_LEN(p) : max_length;
     unsigned long len = 2 * max_len;
     uint8_t encoded_packet[len];
-    Base64Encode((unsigned char*) GET_PKT_DATA(p), max_len, encoded_packet, &len);
+    int rc = Base64Encode((unsigned char*) GET_PKT_DATA(p), max_len, encoded_packet, &len);
+    if (rc != SC_BASE64_OK) {
+        return;
```

Comment:
in other places we don't consider this to be an error, so instead I would suggest something like
```c
int rc = Base64Encode((unsigned char*) GET_PKT_DATA(p), max_len, encoded_packet, &len);
if (rc == SC_BASE64_OK) {
    json_object_set_new(js, "packet", json_string((char *)encoded_packet));
}
```

Reply:
Good point.